### ISSUE TITLE
a11y enhancements of the multiselect

### ISSUE THREAD

**Issue by KimberlyMunoz** [cfpb/consumerfinance.gov#1350]
From https://github.com/cfpb/cfgov-refresh/pull/1249#issuecomment-171749751

> I think having the JavaScript toggle aria-hidden to hide the options will be really useful when this has a list of 50 options. I think have a small snippet of code like <span class="u-visually-hidden" id="selected-fields"> Selected fields: Mortgages, Financial Education</span> that we link with aria-describedby might give some additional context for finding what fields were selected in a huge dropdown.

**Comment by anselmbradford** [comment]
Multiselect appears on the filters on e.g. https://www.consumerfinance.gov/about-us/blog/

![screen shot 2018-04-18 at 5 15 33 pm](https://user-images.githubusercontent.com/704760/38958728-1b2a7e92-432c-11e8-92c8-afaa62a4ce89.png)

**Comment by marteki** [comment]
@anselmbradford I'm scheduled to put https://www.consumerfinance.gov/about-us/blog/ through our accessibility audit by the end of the week. I'll sync up with you to see what (if any) additional fixes/enhancements should be included in this issue.

**Comment by anselmbradford** [comment]
Sounds good! Exciting! 👍

**Comment by GrooveCS** [comment]
Is this still an Open Issue?

**Comment by anselmbradford** [comment]
We have internal a11y issues filed that I think will cover this
